### ISSUE TITLE
visual studio code auto closes when opening

### ISSUE THREAD

**Issue by Osderda** [microsoft/vscode#212503]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.89.1 and other ver.
- OS Version: Win10 22H2 19045
=
After the last update this month, vsCode automatically closes when opening it.
I tried many methods such as reinstalling, installing other versions, deleting the plugin folder, turning off other software running in the background, and I got no results. When it opens, the vscode screen appears and then closes immediately. When I looked at the task manager, I saw that the crash handler was running.

https://github.com/microsoft/vscode/assets/68977883/6b23f9ce-14a6-4114-ab7b-147a36c6cc99

![image](https://github.com/microsoft/vscode/assets/68977883/802ffa0c-f969-4872-8b06-e83da1abdb87)

**Comment by huzaifamk** [comment]
Same thing happening to me on Ubuntu 22.04.

**Comment by Jkimishere** [comment]
Same thing happens on windows 10.
My vscode opens for a few seconds, then closes

on inspecting the logs the closest thing to an error is this:
```
sharedprocess.log
(same thing repeats from above)
.-
2024-05-11 21:01:05.332 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.562 [info] Getting Manifest... ms-python.python
2024-05-11 21:01:06.630 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.630 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.630 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.630 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.630 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.630 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.630 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.630 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.630 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.630 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.630 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.631 [warning] [cmake-tools]: Couldn't find message for key cmake-tools.configuration.cmake.options.advanced.statusBarLength.description.
2024-05-11 21:01:06.656 [info] Waiting for already requested installing extension ms-python.python ms-python.vscode-pylance
```

**Comment by DavudSafarli** [comment]
any fix so far?

**Comment by Jkimishere** [comment]
I encountered a similar issue and was able to resolve it by creating a new directory in a different location. 
In my specific case, I had a Jupyter notebook file open when VSCode closes.
Switching to a new directory (Which closes the Jupyter notebook) seemed to address the problem.
I am not sure if this issue is directly related to the Jupyter notebook, but at least the issue is fixed ¯\_(ツ)_/¯

**Comment by Osderda** [comment]
When I started it with the console window, I saw that it kept a log like this and closed itself.
"Failed to create null handle: The system cannot find the specified file."
I not sure exactly what you're looking for.
![image](https://github.com/microsoft/vscode/assets/68977883/6c5b31d3-adb2-498f-bc18-57c7e71dd3c0)

**Comment by Osderda** [comment]
Oh omg. The system crashed because it couldn't find the driver. 
Solution:
Start cmd administrator and 
paste these two codes in order and try opening VSCode again.
1) `sc create null binPath= C:\Windows\System32\drivers\null.sys type= kernel start= auto error= normal `
OUT> [SC] CreateService SUCCESS 

2) `sc start null`
OUT> SERVICE_NAME: null
        TYPE               : 1  KERNEL_DRIVER
        STATE              : 4  RUNNING
                                (STOPPABLE, NOT_PAUSABLE, IGNORES_SHUTDOWN)
        WIN32_EXIT_CODE    : 0  (0x0)
        SERVICE_EXIT_CODE  : 0  (0x0)
        CHECKPOINT         : 0x0
        WAIT_HINT          : 0x0
        PID                : 0
        FLAGS              :

**Comment by Fuzail98** [comment]
What would be the fix for ubuntu 22.04? I am facing the same issue right now. I tried uninstalling and installing again but to no avail. I even tried deleting the .vscode directory as suggested by one in a stackoverflow (https://stackoverflow.com/questions/75085085/vscode-closes-immediately-after-opening) but no results. I restarted my system as well :P.

If anyone has tried anything to make it work your steps would be appreciated.

Thanks

**Comment by Lecarvalho** [comment]
On linux (I'm using Manjaro) this worked for me:

`sudo snap revert code`

> What would be the fix for ubuntu 22.04? I am facing the same issue right now. I tried uninstalling and installing again but to no avail. I even tried deleting the .vscode directory as suggested by one in a stackoverflow (https://stackoverflow.com/questions/75085085/vscode-closes-immediately-after-opening) but no results. I restarted my system as well :P.
> 
> If anyone has tried anything to make it work your steps would be appreciated.
> 
> Thanks @Fuzail98

**Comment by Fuzail98** [comment]
> On linux (I'm using Manjaro) this worked for me:
> 
> `sudo snap revert code`
> 
> > What would be the fix for ubuntu 22.04? I am facing the same issue right now. I tried uninstalling and installing again but to no avail. I even tried deleting the .vscode directory as suggested by one in a stackoverflow (https://stackoverflow.com/questions/75085085/vscode-closes-immediately-after-opening) but no results. I restarted my system as well :P.
> > If anyone has tried anything to make it work your steps would be appreciated.
> > Thanks @Fuzail98

Thanks @Lecarvalho 
I tried re installing vscode by downloading the deb and the problem went away. Thanks for your response.
